apps/mobile/src/features/send/send-form/hooks/use-send-form-btc.tsx

```diff
@@ -101,7 +101,7 @@ export function useSendFormBtc() {
 
       // Show an error toast here?
       if (!tx) throw new Error('Attempted to generate raw tx, but no tx exists');
-      // eslint-disable-next-line no-console, lingui/no-unlocalized-strings
+      // eslint-disable-next-line no-console
```

Comment:
I think we should probably just get rid of the `console.log`s here

Reply:
Probably out of scope for this PR, I think this is still work in progress too.

Reply:
Just if you are updating the lint, may as well delete the log instead.

Now we've released a beta we should tighten up and get rid of any of this stuff